README.dev.md

```diff
@@ -53,27 +62,28 @@ pytest
 pytest tests/
 ```
 
-## Using `cookiecutter` to generate a new package from the command line
+## Using `copier` to generate a new package from the command line
 
 While making changes to the template, you'll regularly want to verify that the packages generated with the template
 still work. Any easy way to do this is to generate new packages in a temporary directory (which will get removed
 everytime you reboot), for example like so:
 
 ```shell
 # change directory to a new temporary directory
-cd $(mktemp -d --tmpdir cookiecutter-generated.XXXXXX)
+cd $(mktemp -d --tmpdir copier-generated.XXXXXX)
 
-# run cookiecutter with the template to generate a new package
-cookiecutter <path to where your template is>
+# run copier with the template to generate a new package
+copier copy --vcs-ref HEAD  <path/to/project/template> <path/to/destination>
 
 # when it asks you for the GitHub organization, put in your own name;
 # for the other questions, just accept the default
 
 # 'ls' should return just the one directory called 'my-python-project'
 ls 
 ```
+Notice, that the `--vcs-ref HEAD` flag is used to make sure that the current checked out version of a local template are used.
 
-If your Python package was created successfully, `cookiecutter` will point you to a file
+If your Python package was created successfully, `copier` will point you to a file
 (`my-python-project/next_steps.md`) that contains information on next steps.
```

Comment:
Is this still true?

Reply:
As discussed, w leave it for now, this will be addressed in #409